public/static/docs/sidebar.json

```diff
@@ -162,7 +162,7 @@
         "slug": "analytics"
       },
       {
-        "label": "Privacy Policy",
+        "label": "Privacy Policy (Google APIs)",
```

Comment:
> it's probably too long to fit into one line?

@shcheklein nope, it fits 🙂 but that's the reason I shortened the docs title actually.

![image](https://user-images.githubusercontent.com/1477535/74206060-6cc93e00-4c3f-11ea-95c5-02ad059d0168.png)